### PR TITLE
source-coreBOS connector implementation

### PR DESCRIPTION
**Pull request by charity1475**
## What
_Describe what the change is solving_
The connector will help sync data from coreBOS API to other platforms
_It helps to add screenshots if it affects the frontend._

## How
_Describe the solution_
By providing capability to fetch data from any endpoint from the API.

## Pre-merge Checklist
* [ ] _Run integration tests_
* [x] _Publish Docker images_

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368163535111) by [Unito](https://www.unito.io)

### CODE REVIEW

File: airbyte-integrations/connectors/source-corebos/source_corebos/source.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    def read(
+        self, logger: AirbyteLogger, config: json, catalog:json, state: Dict[str, any]
+    ) -> Generator[AirbyteMessage, None, None]:
+
+        retrieve_stream = "retrieve"
+        
+        if retrieve_stream is None:
+            logger.info("No selected stream")
+            return
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-corebos/source_corebos/source.py:92]: you are defining `retrive_stream` above, are you planning to receive this variable as an argument?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@charity1475 thanks for contributing! some comments and can you run `./gradlew format` to apply formating?

File: airbyte-integrations/connectors/source-corebos/source_corebos/source.py (hunk 1 above)
Comment (charity1475) [reply airbyte-integrations/connectors/source-corebos/source_corebos/source.py:92]: nope

**Comment by charity1475** [comment]
I have formatted accordingly. Requesting the merge.

**Comment by charity1475** [comment]
@marcosmarxm

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,7 @@ out
 .DS_Store
 data
 .dockerversions
+airbyte-config/
```
[the remark this answers is not in the document]
Comment (marcosmarxm) [reply .gitignore:9]: @charity1475 can you remove this?

File: airbyte-integrations/connectors/source-corebos/source_corebos/source.py (hunk 1 above)
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-corebos/source_corebos/source.py:92]: i'm we can remove the if block then

File: airbyte-integrations/connectors/source-corebos/source_corebos/source.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            base_url="http://test.coreboscrm.com/denorm/webservice.php"
+            url = base_url
+            logger.info("GET {}".format(url))
+            response = requests.get(url, params=params)
+            if response.status_code == 200:
+                return AirbyteConnectionStatus(status=Status.SUCCEEDED)
+            else:
+                return AirbyteConnectionStatus(
+                    status=Status.FAILED,
+                    message=f"An exception occurred: Status Code: {0}, content: {1}".format(response.status_code, response.content),
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-corebos/source_corebos/source.py:57]: use f-string accordly {0} change for {response.status_code} and {1} for {response.content}

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by charity1475** [comment]
re-requesting review

**Comment by marcosmarxm** [comment]
@charity1475 can you revert commit https://github.com/airbytehq/airbyte/pull/3141/commits/ce3902eb1e9ba092e640a1291178558a9ab543f9 you only need to delete the line you included: `airbyte-config/`.

**Comment by charity1475** [comment]
done

**Pull request closed without merging**